packages/kbn-esql-validation-autocomplete/src/validation/validation.ts

```diff
@@ -358,9 +358,7 @@ function validateFunction(
           messageId: 'wrongArgumentNumber',
           values: {
             fn: astFunction.name,
-            numArgs:
-              refSignature.minParams ??
```

Comment:
Why this removal?

Reply:
Hmmm, not sure. I cherry-picked this from another branch and have no recollection :D 

restored in https://github.com/elastic/kibana/pull/180625/commits/86b42aacfc1c498b869534f3f8d7b4adf39e3291